spec.md

```diff
@@ -390,20 +388,28 @@ Example error response (invalid `id` provided):
 
 #### Collection HTTP API
 
-#### Collection Javascript Client API
-
 ## Resources and Blobs
 
 ### Blob Data Model
 
+A logical unit of data, in transit or at rest.
+(As described in the [W3c FileAPI: Blob Interface](https://w3c.github.io/FileAPI/#blob-section)).
+
+Blob properties:
+
+* byte stream
+* `size` - length of the byte stream in bytes
+  - Note: Although size can be derived from bytes, it's useful to be able to
+    have it up front (for the receiving system to decide to reject an upload
+    based on quota / exceeding max size, etc).
+* `type` (required) (from the IANA mime type registry)
```

Comment:
fyi in DOM FileAPI, it is not really required because it can be ''.

But personally I am willing to make it required here. '' is equivalent to `application/octet-stream`. we can default it to that so folks dont have to write it out, but have it required after that.

Reply:
I like that! let's change it to. like "if missing, defaults to `application/octet-stream`"

Reply:
(changed)